cmake/onnxruntime_java.cmake

```diff
@@ -217,6 +223,7 @@ if (CMAKE_SYSTEM_NAME STREQUAL "Android")
       -b build-android.gradle -c settings-android.gradle
```

Comment:
will the updates to build-android.gradle be in a separate PR?

Reply:
yes, in the packaging pipeline PR